autoload/asyncomplete.vim

```diff
@@ -235,8 +235,18 @@ function! s:notify_sources_to_refresh(ctx, force) abort
         endif
         if l:refresh
             try
-                call asyncomplete#log('core.s:notify_sources_to_refresh', 'completor()', l:source_name, a:ctx)
-                call s:sources[l:source_name].completor(s:sources[l:source_name], a:ctx)
+                let l:ctx = a:ctx
```

Comment:
need to make a copy if we are modifying it since it is in loop. `let l:ctx = copy(a:copy)`